### PR TITLE
Lower default pixel error for 3d tiled scenes to 15 px to match cesium defaults

### PR DESCRIPTION
**Pull request by nyalldawson**

### CODE REVIEW

**Review by wonder-sk — APPROVED** [review]
Wasn't that 16px? :slightly_smiling_face:   (maybe I did not remember that correctly!)

**Comment by nyalldawson** [comment]
I thought it was 15 or 16 🤪 I'll check the recording

File: src/3d/qgstiledscenelayer3drenderer.cpp [hunk 1]
```diff
@@ -86,7 +86,7 @@ void QgsTiledSceneLayer3DRenderer::readXml( const QDomElement &elem, const QgsRe
   mLayerRef = QgsMapLayerRef( elem.attribute( QStringLiteral( "layer" ) ) );
 
   mShowBoundingBoxes = elem.attribute( QStringLiteral( "show-bounding-boxes" ), QStringLiteral( "0" ) ).toInt();
-  mMaximumScreenError = elem.attribute( QStringLiteral( "max-screen-error" ), QStringLiteral( "50.0" ) ).toDouble();
+  mMaximumScreenError = elem.attribute( QStringLiteral( "max-screen-error" ), QStringLiteral( "15.0" ) ).toDouble();
```
Comment (nyalldawson) [inline src/3d/qgstiledscenelayer3drenderer.cpp:89]:
```suggestion
  mMaximumScreenError = elem.attribute( QStringLiteral( "max-screen-error" ), QStringLiteral( "16.0" ) ).toDouble();
```

File: src/3d/qgstiledscenelayer3drenderer.h [hunk 2]
```diff
@@ -114,7 +114,7 @@ class _3D_EXPORT QgsTiledSceneLayer3DRenderer : public QgsAbstract3DRenderer
 
   private:
     QgsMapLayerRef mLayerRef; //!< Layer used to extract mesh data from
-    double mMaximumScreenError = 50.0;
+    double mMaximumScreenError = 15.0;
```
Comment (nyalldawson) [inline src/3d/qgstiledscenelayer3drenderer.h:117]:
```suggestion
    double mMaximumScreenError = 16.0;
```

File: src/app/3d/qgstiledscenelayer3drendererwidget.cpp [hunk 3]
```diff
@@ -26,7 +26,7 @@ QgsTiledSceneLayer3DPropertiesWidget::QgsTiledSceneLayer3DPropertiesWidget( QWid
 {
   setupUi( this );
 
-  mMaxErrorSpinBox->setClearValue( 50 );
+  mMaxErrorSpinBox->setClearValue( 15 );
```
Comment (nyalldawson) [inline src/app/3d/qgstiledscenelayer3drendererwidget.cpp:29]:
```suggestion
  mMaxErrorSpinBox->setClearValue( 16 );
```

**Comment by nyalldawson** [comment]
16 it is!

**Pull request merged**